src/widget/wnumberpos.cpp

```diff
@@ -10,42 +10,16 @@
 
 WNumberPos::WNumberPos(const char* group, QWidget* parent)
         : WNumber(parent),
-          m_dOldPosition(0.0),
-          m_dTrackSamples(0.0),
-          m_dTrackSampleRate(0.0),
-          m_bRemain(false) {
+          m_dOldTime(0.0) {
+    m_pTimeElapsed = new ControlProxy(group, "time_elapsed", this);
```

Comment:
The string literal "time_elapsed" is used in many places. Do we have a place where we could define those literals as constants?

Reply:
Hmm, that's not common practice for ControlObject names, but perhaps it should be...

Reply:
Maybe not now, just noticed how many times it appears in this PR.